gcc/rust/parse/rust-parse.h

```diff
@@ -585,6 +584,12 @@ template <typename ManagedTokenSource> class Parser
   // Construct parser with specified "managed" token source.
   Parser (ManagedTokenSource tokenSource) : lexer (std::move (tokenSource)) {}
 
+  // Parse items without parsing an entire crate. This function is the main
+  // parsing loop of AST::Crate::parse_crate(). It takes a mutable reference
+  // on a vector of errors as parameter in which errors will be appended.
+  std::vector<std::unique_ptr<AST::Item> >
+  parse_items (std::vector<Error> &error_table);
+
```

Comment:
Add a new getter unless one already exists.

std::vector<Error> get_errors()

Reply:
Done in b9f4f1b